packages/common/src/schema.graphql

```diff
@@ -253,6 +253,13 @@ union InsertSupplierInvoiceResponse =
   | NodeError
   | InvoiceNode
 
+type LocationNode {
+  code: String!
+  description: String!
+}
```

Comment:
not using, but added this as we'll need it for supplier invoices anyway, I suspect - actually, I should have added the ID also